README.md

```diff
@@ -21,28 +22,27 @@
-| [terra-clinical-item-view](https://github.com/cerner/terra-clinical/tree/master/packages/terra-clinical-item-view) | [![NPM version](http://img.shields.io/npm/v/terra-clinical-item-view.svg)](https://www.npmjs.org/package/terra-clinical-item-view) | ![Stable](https://img.shields.io/badge/status-Stable-brightgreen.svg) |
-| [terra-clinical-label-value-view](https://github.com/cerner/terra-clinical/tree/master/packages/terra-clinical-label-value-view) | [![NPM version](http://img.shields.io/npm/v/terra-clinical-label-value-view.svg)](https://www.npmjs.org/package/terra-clinical-label-value-view) | ![Stable](https://img.shields.io/badge/status-Stable-brightgreen.svg) |
-| terra-clinical-modal-manager |[![NPM version](http://img.shields.io/npm/v/terra-clinical-modal-manager.svg)](https://www.npmjs.org/package/terra-clinical-modal-manager) | ![Stable](https://img.shields.io/badge/status-Deprecated-lightgrey.svg)|
-| terra-clinical-no-data-view | [![NPM version](http://img.shields.io/npm/v/terra-clinical-no-data-view.svg)](https://www.npmjs.org/package/terra-clinical-no-data-view) | ![Stable](https://img.shields.io/badge/status-Deprecated-lightgrey.svg)|
-| terra-clinical-site |[![NPM version](http://img.shields.io/npm/v/terra-clinical-site.svg)](https://www.npmjs.org/package/terra-clinical-site) | ![Stable](https://img.shields.io/badge/status-Deprecated-lightgrey.svg)|
-| terra-clinical-slide-group |[![NPM version](http://img.shields.io/npm/v/terra-clinical-slide-group.svg)](https://www.npmjs.org/package/terra-clinical-slide-group) | ![Stable](https://img.shields.io/badge/status-Deprecated-lightgrey.svg)|
-
+![Stable](https://badgen.net/badge/status/Stable/green)
+![Beta](https://badgen.net/badge/status/Beta/orange)
+![Deprecated](https://badgen.net/badge/status/Deprecated/grey)
+
+
+| Terra Package      | Version | Status | Dependencies |
+|--------------------|---------|--------|--------------|
+| terra-clinical-action-header | [![NPM version](https://badgen.net/npm/v/terra-clinical-action-header)](https://www.npmjs.org/package/terra-clinical-action-header) | ![Deprecated](https://badgen.net/badge/status/Deprecated/grey)| ![terra-clinical-action-header](https://badgen.net/badge/dependencies/n%2Fa/grey) |
+| terra-clinical-app-delegate |[![NPM version](https://badgen.net/npm/v/terra-clinical-app-delegate)](https://www.npmjs.org/package/terra-clinical-app-delegate) | ![Deprecated](https://badgen.net/badge/status/Deprecated/grey)| ![terra-clinical-app-delegate](https://badgen.net/badge/dependencies/n%2Fa/grey) |
+| terra-clinical-application |[![NPM version](https://badgen.net/npm/v/terra-clinical-application)](https://www.npmjs.org/package/terra-clinical-application) | ![Deprecated](https://badgen.net/badge/status/Deprecated/grey)| [![terra-clinical-application](https://badgen.net/david/dep/cerner/terra-clinical/packages/terra-clinical-application)](https://david-dm.org/cerner/terra-clinical?path=packages/terra-clinical-application) |
```

Comment:
For the dependency badge for packages that have been deprecated, I went with using the following 

`![terra-clinical-application](https://badgen.net/badge/dependencies/n%2Fa/grey)`